articles/sso/current/single-page-apps-sso.md

```diff
@@ -171,7 +171,7 @@ setInterval(function() {
   // if the token is not in local storage, there is nothing to check (i.e. the user is already logged out)
   if (!localStorage.getItem('userToken')) return;
 
-  auth0.getSSOData(function (err, data) {
+  auth0.checkSession(function (err, data) {
     // if there is still a session, do nothing
     if (err || (data && data.sso)) return;
```

Comment:
I think this example is not valid anymore. Does checkSession returns a .sso property?

Anyway let's wait until we reimplement getSSOData